ci-operator/step-registry/mco/conf/day2/destroy-custom-mcp/OWNERS

```diff
@@ -0,0 +1,16 @@
+approvers:
```

Comment:
@JianLi-RH I think we can replace the lists with our names.

Reply:
as well as the owners for `mco-conf-day2-create-custom-mcp` ?

Reply:
I suggest not to touch with `mco-conf-day2-create-custom-mcp`.

Reply:
also updated ci-operator/step-registry/openshift/upgrade/qe/test/canary/OWNERS